teaser/logic/buildingobjects/calculation/one_element.py

```diff
@@ -932,15 +932,15 @@ def _fill_zone_lists(self):
                 self.shading_g_total.append(
```

Comment:
A few lines above, line 920, has this to be set to 1 there? Well, it probably doesn`t matter because there is no window.
And has shading_max_irr to be added there as well?

Reply:
I'll check the default values (i.e. no shading) on monday